docker-for-mac/apple-m1.md

```diff
@@ -57,11 +57,24 @@ The following issues are not expected to be resolved in the final GA build for A
 
    We are still gathering data and testing alternate kernel versions.
 
+### Fixes since Docker Desktop RC 1
+
+- Inter-container HTTP and HTTPS traffic is now routed correctly. Previously it was accidentally forwarded to an HTTP proxy on the host. Fixes [docker/for-mac#5476](https://github.com/docker/for-mac/issues/5476).
```

Comment:
Omit middle sentence. (Focus on the customer symptom, not the cause of the bug).